README.md

```diff
@@ -16,44 +16,51 @@ random: use random eligibility for data request witnessing instead of stake rati
 
 Options:
 
-- --stakers
-    - explanation: number of total stakers to simulate
-    - default: 100
-- --total-staked
-    - explanation: total amount staked
-    - default: 100000000
-- --distribution
-    - explanation: defines the distribution of the amount each staker (relative to the total amount)
-    - default: random
-    - allowed values: random, uniform, beta, exponential, gamma
-- --whales
-    - explanation: adds a number of whale stakers to the staking distribution
-    - default: 0
-- --whales-stake
-    - explanation: percentage of the total staked to divide uniformly over the number of whales
-    - default: 0
+- --logging
+    - explanation: set the logging level
+    - default: info
+    - allowed values: debug, info, warning, error, critical
 - --epochs
     - explanation: the number of epochs for which to run the mining simulator
     - default: 100000
-- --replication
-    - explanation: set the replication factor to approximate the number of blocks generated per epoch
-    - default: 16
+- --block-reward
+    - explanation: set the block reward for an accepted block
+    - default: 250
+- --max-staking-txs-per-block
+    - explanation: max number of staking transactions allowed per block
+    - default: 32
+- --mining-eligibility
+    - explanation: allows setting different strategies for miner selection
+    - default: vrf-stake-adaptative
+    - allowed values: vrf-stake-adaptative, vrf-stake, module-stake, modulo-slot, vrf-stake-adaptive
```

Comment:
vrf-stake-adaptative and vrf-stake-adaptive?

Reply:
fixed